guides/common/modules/proc_creating-a-custom-certificate.adoc

```diff
@@ -0,0 +1,106 @@
+[id="creating-a-custom-certificate_{context}"]
+
+= Creating a Custom SSL Certificate for {ProductName}
+
+ifeval::["{context}" == "{project-context}"]
+Use this procedure to create a custom SSL certificate for {ProductName}.
+If you already have a custom SSL certificate for {ProductName}, skip this procedure.
+endif::[]
+
+ifeval::["{context}" == "{smart-proxy-context}"]
+On {ProjectServer}, create a custom certificate for your {ProductName}.
+If you already have a custom SSL certificate for {ProductName}, skip this procedure.
+endif::[]
+
+.Procedure
+
+. To store all the source certificate files, create a directory that is accessible only to the `root` user.
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# mkdir /root/{context}_cert
```

Comment:
I wonder if we should recommend `/etc/pki` as a base directory? @ekohl @evgeni

Reply:
If it'd be underneath `/etc/pki/katello` it would even end up in the backup… ;-)

Reply:
That's quite easy to mess up. Relying on `/root` being shielded has the least chance of messing up from a security point of view, because it already has the permissions set before you even start.

Reply:
Exactly what I wrote in https://github.com/theforeman/foreman-documentation/pull/2487#discussion_r1352530581. Short term go for `/root`. Long term we should move to some other location but that requires more work.

Reply:
Ohk. I was confused between both of your suggestions. Applied :+1: